src/services/findAllReferences.ts

```diff
@@ -218,8 +211,8 @@ namespace ts.FindAllReferences {
         return { fileName, span };
     }
 
-    function getTextSpan(node: Node): TextSpan {
-        let start = node.getStart();
+    function getTextSpan(node: Node, sourceFile = node.getSourceFile()): TextSpan {

```

Comment:
no need for the initializer, you want to pass the sourceFile if you have it, otherwise, getStart will walk up and find it, if it needs it.